lib/quibble.js

```diff
@@ -161,3 +161,12 @@ var hackErrorStackToGetCallerFile = function (includeGlobalIgnores) {
     Error.prepareStackTrace = originalFunc
   }
 }
+
+var esmPath
+try {
+  esmPath = resolve.sync('esm/esm', { paths: module.parent.paths })
```

Comment:
Does `module.parent.paths` do what we want it to do in cases where quibble is a second/third/fourth degree transitive dependency for a user? Would we want to resolve relative to `process.cwd()` or some other top-level?

Reply:
Totally! I will set up an example project for transitive dependencies and see how this technique fares. I imagine we will need to walk the `module.parent` tree or use a different technique altogether.